templates/azure/shared/_how-ubuntu-pro-helps-you.html

```diff
@@ -0,0 +1,82 @@
+<div class="row">
+  <div class="col-12">
+    <h2>How Ubuntu Pro helps you</h2>
+  </div>
+
+  <div class="col-4">
+    {{ image (
+      url="https://assets.ubuntu.com/v1/ec3f53f7-extended-security-maintenance.svg",
+      alt="",
+      width="60",
+      height="60",
+      hi_def=True,
+      loading="lazy",
+      attrs={"class": "u-hide--small"}
+      ) | safe
+    }}
+
+    <p>Ubuntu Pro enables instant up-to-date security by default with security patches for thousands of open source applications</p>
+  </div>
+
+  <div class="col-4">
+    {{ image (
+      url="https://assets.ubuntu.com/v1/4f70dabc-Kernel.svg",
+      alt="",
+      width="52",
+      height="60",
+      hi_def=True,
+      loading="lazy",
+      attrs={"class": "u-hide--small"}
+      ) | safe
+    }}
+
+    <p>Azure-optimised kernel for improved performance</p>
+  </div>
+
+  <div class="col-4">
+    {{ image (
+      url="https://assets.ubuntu.com/v1/1d5f293c-compliance.svg",
+      alt="",
+      width="105",
+      height="60",
+      hi_def=True,
+      loading="lazy",
+      attrs={"class": "u-hide--small"}
+      ) | safe
+    }}
+
+    <p>Full compliance checklist. FIPS, HIPAA, PCI, ISO and DISA-STIGs.</p>
```

Comment:
Tiny nitpick, this seems to be the only one ending in `.`, it matches the copy so feel free to merge anyway.

Reply:
Oh, good catch - I'll update it (and the doc) to:
```suggestion
    <p>Full compliance checklist: FIPS, HIPAA, PCI, ISO and DISA-STIGs</p>
```